the-blue-alliance-ios/View Elements/Match/MatchSummaryView.swift

```diff
@@ -144,30 +157,67 @@ class MatchSummaryView: UIView {
     private func teamLabel(for teamKey: String, baseTeamKey: String?, dq: Bool) -> UILabel {
         let text: String = "\(Team.trimFRCPrefix(teamKey))"
         let isBold: Bool = (teamKey == baseTeamKey)
-
+        
         return label(text: text, isBold: isBold, isStrikethrough: dq)
     }
+    
+    private func teamButton(for teamKey: String, baseTeamKey: String?, dq: Bool) -> UIButton {
+        let text: String = "\(Team.trimFRCPrefix(teamKey))"
+        let isBold: Bool = (teamKey == baseTeamKey)
+
+        return button(text: text, isBold: isBold, isStrikethrough: dq)
+    }
+    
+    private func button(text: String, isBold: Bool, isStrikethrough: Bool = false, isTeam: Bool = true) -> UIButton {
+        let newButton = UIButton(type: .system)
+        newButton.setTitle(text, for: [])
+        
+        // set color of the label to black like default
+        newButton.setTitleColor(.primaryBlue, for: .normal)
+        
+        if isTeam {
```

Comment:
Can we combine this with the next line? We should be able to say

`if isTeam, if let teamNumber = Int(text) {`